### ISSUE TITLE
How to perform OR query

### ISSUE THREAD

**Issue by vaibhav-jain** [carltongibson/django-filter#514]
```
class MyFilter(django_filters.FilterSet):
    class Meta:
        model = MyModel
        fields = {
            'field': ['exact',  'isnull'],
        }
```

Now How do I filter this `?field=1&field__isnull=True`  ???

**Comment by carltongibson** [comment]
http://stackoverflow.com/questions/39929084/how-to-perform-or-query-using-django-filter

**Comment by vaibhav-jain** [comment]
I have posted my solution on the SO post referenced by @carltongibson above. @carltongibson thanks for this awesome app.

**Comment by eamigo86** [comment]
Hi @vaibhav-jain  , you can update the [stackoverflow](http://stackoverflow.com/questions/39929084/how-to-perform-or-query-using-django-filter) solution for this issue, which appears to be obsolete.

**Comment by rpkilby** [comment]
HI @eamigo86, the only change necessary should be that `action` was renamed to `method`. I've submitted an edit to the stackoverflow answer.

In prior versions we took care to provide deprecation warnings where possible. If you are running into upgrade issue, try running your tests with the -Werror option, then upgrade incrementally as those issues are fixed. I'd also point you to the [migration docs](http://django-filter.readthedocs.io/en/master/guide/migration.html), which contains more details.
